backport

```diff
@@ -157,4 +163,5 @@ fi
 log_info "Creating new PR..."
 gh pr create --base "$BASE_BRANCH" --title "$NEW_COMMIT_MSG" \
   --body "Backport of $ORIGINAL_PR_URL$(echo -e "\n\n")$PR_BODY" \
-  --assignee "@me" $REVIEWERS_ARG "${LABELS_ARG[@]}" --milestone "$SUFFIX" --web
+  --assignee "${GITHUB_ACTOR:-@me}" $REVIEWERS_ARG "${LABELS_ARG[@]}" --milestone "$SUFFIX" \
+  $( [ "$CI" != true ] && echo "--web" )
```

Comment:
missing new line

Reply:
> missing new line

Not sure what you mean, sorry?

Reply:
Could you add a newline at the end of the file? ;) 

<img width="392" alt="image" src="https://github.com/hazelcast/backport/assets/1242724/bc7604b5-fb36-4380-9010-b499fe804489">

Reply:
> Could you add a newline at the end of the file? ;)
> 
> <img alt="image" width="392" src="https://private-user-images.githubusercontent.com/1242724/317513977-bc7604b5-fb36-4380-9010-b499fe804489.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1Nzk2NjYsIm5iZiI6MTcxMTU3OTM2NiwicGF0aCI6Ii8xMjQyNzI0LzMxNzUxMzk3Ny1iYzc2MDRiNS1mYjM2LTQzODAtOTAxMC1iNDk5ZmU4MDQ0ODkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjdUMjI0MjQ2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2YyYmY3YjdlMjUyODVkOTIyNzE2NjJkZTFiMzU5M2QzMzNmMzNlNDYwMWM2MjA4Yzc2ZGFjZGYzNTdlZjliMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GQeJz-pjuIjICCHii6WkWVMVuOFjnR7LKLi456spZfU">


There is a blank line if you download it, it's just not showing in the GitHub preview or diff for some reason:

![image](https://github.com/hazelcast/backport/assets/8626721/5d5d335c-8c9d-443d-9bff-3b8bb5f5d097)


The same happens in the[ original one too](https://raw.githubusercontent.com/hazelcast/backport/master/backport).